src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -119,8 +118,10 @@ class DepositServlet(app: EasyDepositApiApp)
     (for {
       uuid <- getUUID
       path <- getPath
-      depositFiles <- app.getDepositFiles(user.id, uuid, path)
-    } yield Ok(body = toJson(depositFiles))
+      dataFiles <- app.getDataFiles(user.id, uuid)
```

Comment:
This type of logic (deciding between directory? or single file?) should really be handled in the application object.

Reply:
I have now moved the distinction between directories and files into `EasyDepositApiApp`

Reply:
@vesaakerman @jo-pol OK, that looks better. `getDataFiles` can now be made private by the way. It is however a pity that the return type of getFileInfo is now `Try[Object]`.  Here it *would* make sense to have sepate `listFileInfos` and `getFileInfo` methods.